These targets are dead code but they aren't available with system abseil
Based on https://build.opensuse.org/package/view_file/openSUSE:Factory:ARM/nodejs-electron

--- third_party/abseil-cpp/BUILD.gn	2023-12-04 22:46:30.787796700 +0100
+++ third_party/abseil-cpp/BUILD.gn	2023-12-06 14:41:10.534427246 +0100
@@ -61,8 +61,6 @@
     "//third_party/abseil-cpp/absl/base",
     "//third_party/abseil-cpp/absl/base:config",
     "//third_party/abseil-cpp/absl/base:core_headers",
-    "//third_party/abseil-cpp/absl/base:nullability",
-    "//third_party/abseil-cpp/absl/base:prefetch",
     "//third_party/abseil-cpp/absl/cleanup",
     "//third_party/abseil-cpp/absl/container:btree",
     "//third_party/abseil-cpp/absl/container:fixed_array",
@@ -77,24 +75,17 @@
     "//third_party/abseil-cpp/absl/functional:any_invocable",
     "//third_party/abseil-cpp/absl/functional:bind_front",
     "//third_party/abseil-cpp/absl/functional:function_ref",
-    "//third_party/abseil-cpp/absl/functional:overload",
     "//third_party/abseil-cpp/absl/hash",
-    "//third_party/abseil-cpp/absl/log:absl_check",
-    "//third_party/abseil-cpp/absl/log:absl_log",
-    "//third_party/abseil-cpp/absl/log:die_if_null",
     "//third_party/abseil-cpp/absl/memory",
     "//third_party/abseil-cpp/absl/meta:type_traits",
     "//third_party/abseil-cpp/absl/numeric:bits",
     "//third_party/abseil-cpp/absl/numeric:int128",
     "//third_party/abseil-cpp/absl/random",
-    "//third_party/abseil-cpp/absl/random:distributions",
     "//third_party/abseil-cpp/absl/status",
     "//third_party/abseil-cpp/absl/status:statusor",
     "//third_party/abseil-cpp/absl/strings",
     "//third_party/abseil-cpp/absl/strings:cord",
-    "//third_party/abseil-cpp/absl/strings:has_ostream_operator",
     "//third_party/abseil-cpp/absl/strings:str_format",
-    "//third_party/abseil-cpp/absl/strings:string_view",
     "//third_party/abseil-cpp/absl/synchronization",
     "//third_party/abseil-cpp/absl/time",
     "//third_party/abseil-cpp/absl/types:optional",
@@ -193,33 +184,18 @@
         "absl/algorithm:algorithm_test",
         "absl/algorithm:container_test",
         "absl/base:config_test",
-        "absl/base:nullability_test",
         "absl/base:prefetch_test",
         "absl/cleanup:cleanup_test",

         # TODO(mbonadei): Fix issue with EXPECT_DEATH and uncomment.
         # "absl/container:btree_test",
         "absl/container:common_policy_traits_test",
-        "absl/container:fixed_array_test",
-        "absl/container:flat_hash_set_test",
         "absl/container:inlined_vector_test",
         "absl/container:node_slot_policy_test",
-        "absl/container:raw_hash_set_allocator_test",
         "absl/container:sample_element_size_test",
-        "absl/crc:crc32c_test",
-        "absl/crc:crc_cord_state_test",
-        "absl/crc:crc_memcpy_test",
-        "absl/crc:non_temporal_memcpy_test",
-        "absl/debugging:stacktrace_test",
-        "absl/flags:flag_test",
         "absl/functional:any_invocable_test",
-        "absl/functional:function_ref_test",
-        "absl/functional:overload_test",
-        "absl/hash:hash_instantiated_test",
         "absl/hash:hash_test",
         "absl/hash:low_level_hash_test",
-        "absl/log:absl_check_test",
-        "absl/log:absl_log_basic_test",
         "absl/log:die_if_null_test",
         "absl/log:flags_test",
         "absl/log:globals_test",
@@ -231,18 +207,13 @@
         "absl/log:log_streamer_test",
         "absl/log:scoped_mock_log_test",
         "absl/log:stripping_test",
-        "absl/log/internal:fnmatch_test",
         "absl/log/internal:stderr_log_sink_test",
         "absl/memory:memory_test",
         "absl/meta:type_traits_test",
-        "absl/numeric:int128_test",
         "absl/profiling:exponential_biased_test",
         "absl/profiling:periodic_sampler_test",
-        "absl/status:status_test",
         "absl/status:statusor_test",
         "absl/strings:ascii_test",
-        "absl/strings:char_formatting_test",
-        "absl/strings:charset_test",
         "absl/strings:cord_buffer_test",
         "absl/strings:cord_data_edge_test",
         "absl/strings:cord_rep_btree_navigator_test",
@@ -255,18 +226,11 @@
         "absl/strings:cordz_test",
         "absl/strings:cordz_update_scope_test",
         "absl/strings:cordz_update_tracker_test",
-        "absl/strings:damerau_levenshtein_distance_test",
-        "absl/strings:has_absl_stringify_test",
-        "absl/strings:has_ostream_operator_test",
         "absl/strings:match_test",
         "absl/strings:str_replace_test",
         "absl/strings:string_view_test",
-        "absl/synchronization:kernel_timeout_internal_test",
-        "absl/synchronization:waiter_test",
-        "absl/time:time_test",
         "absl/types:optional_test",
         "absl/types:variant_test",
-        "absl/utility:if_constexpr_test",
         "//third_party/googletest:gtest_main",
       ]
     }

--- third_party/fuzztest/BUILD.gn	2023-12-06 14:43:06.483268503 +0100
+++ third_party/fuzztest/BUILD.gn	2023-12-06 14:43:38.348117092 +0100
@@ -259,7 +259,6 @@
     # destined for ClusterFuzz.
     "//testing/libfuzzer:fuzzing_engine",
     "//third_party/abseil-cpp/absl/flags:flag",
-    "//third_party/abseil-cpp/absl/random:bit_gen_ref",
   ]

   public_deps = [
